src/super_gradients/training/transforms/pipeline_adaptors.py

```diff
@@ -0,0 +1,37 @@
+from typing import Callable
+from abc import abstractmethod, ABC
+import numpy as np
+
+
+class TransformsPipelineAdaptorBase(ABC):
+    def __init__(self, composed_transforms: Callable):
+        self.composed_transforms = composed_transforms
+
+    @abstractmethod
+    def __call__(self, sample, *args, **kwargs):
+        raise NotImplementedError
+
+    @abstractmethod
+    def prep_for_transforms(self, sample):
+        raise NotImplementedError
+
+    @abstractmethod
+    def post_transforms_processing(self, sample):
+        raise NotImplementedError
+
+
+class AlbumentationsAdaptor(TransformsPipelineAdaptorBase):
+    def __init__(self, composed_transforms: Callable):
+        super(AlbumentationsAdaptor, self).__init__(composed_transforms)
```

Comment:
I think this is where we should instantiate A.Compose as in addition to transforms this class allow to specify bbox_params which would be necessary for detection task.

```
transform = A.Compose(
    [A.RandomSizedBBoxSafeCrop(width=448, height=336, erosion_rate=0.2)],
    bbox_params=A.BboxParams(format='coco', min_area=0.5, label_fields=['category_ids']),
)
```

Reply:
I actually disagree, since this would limit the users to use Compose, whilst actually they can actually pass something else that does not need to be wrapped (see my use of ALBUMENTATIONS_COMP_TRANSFORMS).

Reply:
A top-level Compose class is not only a container for augmentations, it handles way more on top of that. For detection, users will have to pass bbox_params to Compose to specify bounding box format that comes in. How this would look in YAML config for that case?

Reply:
In exactly the same way we pass any other parameter to a transform in the YAML file, we just need to extend the support for A.BboxParams inside the factory.

Reply:
transform:
  Albumentations:
    Compose:
      transforms:
        - RandomSizedBBoxSafeCrop: ...
      bbox_params:
        BboxParams: ...